tests/typecheck_examples.tests

```diff
@@ -0,0 +1,1007 @@
+examples/dictionary/dictSuggest.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/dictionary/dictSuggestLite.links
+examples/dictionary/dictSuggestLite.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/dictionary/dictSuggestUpdate.links
+examples/dictionary/dictSuggestUpdate.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/chatserver/chatClient.links
+examples/distribution/chatserver/chatClient.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config --path=examples/distribution/chatserver:examples/distribution/
+
+Typecheck example file examples/distribution/chatserver/chatServer.links
+examples/distribution/chatserver/chatServer.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config --path=examples/distribution/chatserver:examples/distribution/
+
+Typecheck example file examples/distribution/chatserver/chatSessions.links
+examples/distribution/chatserver/chatSessions.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientProcessOnServer.links
+examples/distribution/clientProcessOnServer.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientProcessOnServer2.links
+examples/distribution/clientProcessOnServer2.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientToClient.links
+examples/distribution/clientToClient.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientToClientDeleg.links
+examples/distribution/clientToClientDeleg.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientToClientViaServerAP.links
+examples/distribution/clientToClientViaServerAP.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientToServerDeleg.links
+examples/distribution/clientToServerDeleg.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/serverToClient.links
+examples/distribution/serverToClient.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/serverToClient2.links
+examples/distribution/serverToClient2.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/simpleServerAP.links
+examples/distribution/simpleServerAP.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+ignore : broken example
```

Comment:
You may want to check with @SimonJF whether the "broken" distribution examples are indeed broken. Maybe they just need to be deleted...

Reply:
@SimonJF could you have a look at this and determine which of the session types examples should be working?

Reply:
Good idea. Planning to spend a day hacking Links on Friday, so I'll take a look at the session types examples to tidy up / determine which should be retained.